content/800-data-platform/100-accelerate/580-local-development.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Development workflow'
+metaTitle: 'Accelerate: Development environment'
```

Comment:
this page is very specifically about "local development", why not make that the title?

```suggestion
title: 'Local development'
metaTitle: 'Accelerate: Local development'
```

Reply:
The original title felt a bit too long. Your suggestion is much better 🙇 , thank you!